plugins/ProfessionalServices/ProfessionalServices.php

```diff
@@ -13,14 +13,14 @@
 class ProfessionalServices extends \Piwik\Plugin
 {
     /**
-     * @see Piwik\Plugin::registerEvents
+     * @see \Piwik\Plugin::registerEvents
      */
     public function registerEvents()
     {
         return array(
             'AssetManager.getStylesheetFiles' => 'getStylesheetFiles',
             'Request.getRenamedModuleAndAction' => 'renameProfessionalServicesModule',
-            'Template.afterGoalConversionOverviewReport' => 'getGoalOverviewPromo',
+            'Template.afterGoalConversionOverviewReport' => array('function' => 'getGoalOverviewPromo', 'after' => true),
```

Comment:
we should always show the promo at the end in case other plugins want to add content to this event